tests/component_handlers/kafka_connect/test_connect_wrapper.py

```diff
@@ -198,6 +198,24 @@ def test_should_raise_connector_not_found_when_getting_connector(
             f"The named connector {connector_name} does not exists."
         )
 
+    def test_should_raise_connector_name_do_not_match(self):
+        configs = {
+            "connector.class": "org.apache.kafka.connect.file.FileStreamSinkConnector",
+            "tasks.max": "1",
+            "topics": "test-topic",
+            "name": "OtherWrongName",
+        }
+        with pytest.raises(ValueError) as value_error:
+            self.connect_wrapper.validate_connector_config(
+                "FileStreamSinkConnector",
+                kafka_connect_config=KafkaConnectConfig(**configs),
+            )
+
+        assert (
+            str(value_error.value)
+            == "Connector name should be the same as component name"
+        )
```

Comment:
indent this to be contained inside the context manager

Reply:
ok, good point!
but wouldn't the test fail anyways then?

Reply:
yes, it would fail, but we couldn't be sure that the message is the message we want. So, if we raise an exception that is the same type but different message, it would pass the test.

Reply:
you're right! I had no idea, thanks!